userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -92,6 +103,10 @@ _Example:_
 
 |===
 
+External Payment Method:: A built-in payment method that can be used to track payments that occur outside of Kill Bill (example: check payments).
+
+Event:: A message exposed by Kill Bill core components when a certain occurrence takes place in the system. For example, when a subscription is created, Kill Bill generates an event corresponding to it.
```

Comment:
I guess I don't see the reason for the "message" part of the definition. To me (as a non-technical person), it sounds like we're saying an event occurs, and Kill Bill exposes a message associated with the event. Is that the case? If so, do we need that level of detail?

Reply:
It's difficult... perhaps we just remove it. This is not really a billing concept anyways.

Reply:
Deleted `Event` definition for now.